docs/agents/vantage_pro2.rst

```diff
@@ -0,0 +1,80 @@
+.. highlight:: rst
+
+.. _vantagepro2:
+
+=============
+Vantage Pro2 Agent
+=============
+
+The Davis Instruments Vantage Pro2 is a weather system + monitor used to 
+acquire and readout weather data. The Vantage Pro2 monitor is connected to
+the labratory computer via usb cable and data is sent though that connection.
+
+.. argparse::
+    :filename: ../agents/vantagePro2_agent/vantage_pro2_agent.py
+    :func: make_parser
+    :prog: python3 vantage_pro2_agent.py
+
+Description
+-----------
+Out of the box, one just needs to connect the weather station to the 
+Vantage Pro2 monitor, and the monitor to the lab computer via usb. As long as
+the Vantage Pro2 monitor is on, the intitialize and data acquisition tasks 
+should work "out of the box", as it where.
+
+The Vantage Pro2 monitor and weather station records 
```

Comment:
Seems like you were going to give a description of the data fields recorded by the weather station